config/sequelize.js

```diff
@@ -4,6 +4,6 @@ module.exports = {
     password: process.env.DB_PASSWORD,
     database: process.env.DB_DATABASE,
     host: process.env.DB_HOSTNAME,
-    dialect: 'mysql',
+    dialect: process.env.DB_DIALECT,
```

Comment:
Yeah this actually made your server not run because the .env-sample doesn't include `DB_DIALECT`, not sure why you changed working server code.